src/components/pages/Home/index.vue

```diff
@@ -0,0 +1,15 @@
+<template lang="pug">
+  .home
+    h1 HOME
```

Comment:
This is OK but it's better to use pipe notation for plain texts in pug. It's used this way:
```pug
  h1
    | HOME
```